torch/_subclasses/fake_tensor.py

```diff
@@ -566,24 +566,20 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         flat_arg_tensors = [
             i for i in tree_flatten((args, kwargs))[0] if isinstance(i, FakeTensor)
         ]
-        has_symbolic_sizes = any([i.has_sym_ints for i in flat_arg_tensors])
+        flat_symints = [
+            i
+            for i in tree_flatten((args, kwargs))[0]
+            if isinstance(i, torch._C.SymIntNode)
```

Comment:
can someone pleaaase give this a better name lol. torch.SymIntNode or torch.SymInt

Reply:
where do I register it to give it a better name?

Reply:
I think monkeying around with `torch/__init__.py` so it gets set there is the idea

Reply:
If you want printing to also look nice, you can update the `__module__` field to match.